style.css

```diff
@@ -0,0 +1,85 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@300&display=swap");
+
+body {
+  background-image: url(header-shapes\ mobile.png);
+  font-family: "Poppins", sans-serif;
+  background-repeat: no-repeat;
+}
+
+header {
+  display: flex;
+  margin: 5px;
+  padding: 5px;
+}
+
+.Logo h4 {
+  color: #6070ff;
+  margin: 5px;
+  padding: 10px;
+}
+
+.Menu-bar {
+  float: right;
+  position: absolute;
+  top: 10px;
+  right: 20px;
+}
+
+.Menu-bar li {
+  list-style: none;
+  width: 20px;
+  height: 3px;
+  background-color: #6070ff;
+  margin: 3px;
+}
+
+.Header-main {
+  width: 462px;
+  height: 587.15px;
+  left: -34px;
+  top: 104px;
+}
```

Comment:
- [ ] If you carefully revisit the [Figma template design], you can notice that there is a `background image`. I therefore, suggest that you consider applying the background image to stick to the Figma design and also have a professional portfolio with a beautiful user interface 😎 ☺️ .